cdc/log.cc

```diff
@@ -416,6 +416,194 @@ utils::UUID generate_timeuuid(api::timestamp_type t) {
     return utils::UUID_gen::get_random_time_UUID_from_micros(t);
 }
 
+struct atomic_column_update {
+    column_id id;
+    atomic_cell cell;
+};
+
+struct nonatomic_column_deletion {
+    column_id id;
+};
+
+struct nonatomic_column_update {
+    column_id id;
+    std::vector<std::pair<bytes, atomic_cell>> cells;
+};
+
+struct static_row_update {
+    gc_clock::rep ttl;
```

Comment:
Why `rep` and not `time_point`?

Reply:
`ttl` is a `duration`, not a `time_point`.
I used `rep` because we need to call `count()` eventually, but ok, `duration` will be better.

Reply:
Ah, I used `rep` so I don't need to write `gc_clock::duration(0)` in places like this:
```
                        view.is_live_and_has_ttl() ? view.ttl().count() : 0
```
this would instead have to be
```
view.is_live_and_has_ttl() ? view.ttl() : gc_clock::duration(0)
```

Reply:
:) Personally I'd still use `duration`, but I won't insist.

Reply:
I will change it to `duration`, using `rep` is an abstraction leak